clang/test/CodeGenSYCL/intel-fpga-no-global-work-offset.cpp

```diff
@@ -1,49 +0,0 @@
-// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -triple spir64-unknown-unknown -disable-llvm-passes -sycl-std=2017 -emit-llvm -o - %s | FileCheck %s
```

Comment:
I don't think that this and many other tests should be removed: even though they pass `-sycl-std=2017`, that's likely a legacy - they do not test SYCL 1.2.1, but instead they test some of our custom attributes and we should uplift the SYCL version used instead of removing those tests